grafana-plugin/src/pages/incident/Incident.tsx

```diff
@@ -214,6 +213,14 @@ class IncidentPage extends React.Component<IncidentPageProps, IncidentPageState>
     const integration = store.alertReceiveChannelStore.getIntegration(incident.alert_receive_channel);
 
     const showLinkTo = !incident.dependent_alert_groups.length && !incident.root_alert_group && !incident.resolved;
+
+    const integrationNameWithoutEmojies =
+      incident.alert_receive_channel.verbal_name.indexOf(':') >= 0
+        ? incident.alert_receive_channel.verbal_name.substring(
+            0,
+            incident.alert_receive_channel.verbal_name.indexOf(':')
+          )
+        : incident.alert_receive_channel.verbal_name.replace(/\p{Emoji}/gu, '');
```

Comment:
hmm, why do you decide to get rid of emojies in this place only? in other places we will still see emojies, looks inconsistent

Reply:
It is because of advice from @raphael-batte.
 Could you comment on it?

Reply:
- The idea behind it was to make labels visually less noisy. 
- Second reason is that some emoji have empty space inside a symbol. This inaccuracy in margins too much visible when we place emoji inside the label.

Reply:
Ok, lets leave them in labels and collect feedback from users.

Reply:
Emojies are back! :)